src/test/java/com/puppycrawl/tools/checkstyle/internal/AllTestsTest.java

```diff
@@ -40,6 +40,8 @@
  * AllTestsTest.
  *
  * @noinspection ClassIndependentOfModule
+ * @noinspectionreason architecture of test modules requires this structure
+ *
```

Comment:
Remove extra empty line as it is the end of the javadoc.

Reply:
Done.